docs/source/access-data/tutorials/initial-setup.rst

```diff
@@ -3,194 +3,189 @@ Initial Setup
 
-    # install the CLI
-    curl https://sdk.cloud.google.com | bash  # Linux and MacOS:
-        # Windows: see https://cloud.google.com/sdk/docs/downloads-interactive#windows
-    # follow the directions
+    .. code-block:: console
 
-    # open a new terminal or restart your shell
-    # exec -l $SHELL
+        $ # insert your project ID from step 1:
+        $ PROJECT_ID=my-pgb-project
 
-    # connect to a Google account
-    gcloud init
-    gcloud auth login
-    # this will open a browser and prompt you for authorization. follow the instructions
+        $ # insert the path to the key file you just downloaded
+        $ KEY_PATH=/local/path/to/GCP_auth_key.json
 
-Create a new GCP project and set it as your local default.
+        $ # set the environment variables
+        $ export GOOGLE_CLOUD_PROJECT=$PROJECT_ID
+        $ export GOOGLE_APPLICATION_CREDENTIALS=$KEY_PATH
 
-.. code:: bash
 
-    gcloud projects create $PROJECT_ID
-    gcloud config set project $PROJECT_ID
+Step 3: Enable and install APIs
+---------------------------------
 
-Create a service account, give it owner permissions, and download an authentication key file.
+**On GCP:**
 
-.. code:: bash
+    Enable the desired APIs in your project.
+    Below are direct links to the APIs needed for our tutorials.
+    You only need to enable the ones you want to use
+    (e.g., to complete the Pub/Sub Streams tutorial, enable the Pub/Sub API).
+    Follow the link below, make sure your project is selected in the dropdown menu
+    at the top of the webpage, then click "Enable".
 
-    gcloud iam service-accounts create $SA_NAME
-    gcloud projects add-iam-policy-binding $PROJECT_ID \
-        --member="serviceAccount:$SA_EMAIL" \
-        --role="roles/owner"
-    gcloud iam service-accounts keys create $KEY_PATH --iam-account=$SA_EMAIL
+    - `BigQuery
+      <https://console.cloud.google.com/apis/library/bigquery.googleapis.com>`__
+      (databases)
+    - `Cloud Storage
+      <https://console.cloud.google.com/apis/library/storage-component.googleapis.com>`__
+      (file storage)
+    - `Pub/Sub <https://console.cloud.google.com/apis/library/pubsub.googleapis.com>`__
+      (message streams)
 
-Set local environment variables that will be used by API calls.
+    Experienced users may wish to enable other APIs.
+    This can be done from the
+    `API Library <https://console.cloud.google.com/apis/library>`__:
+    search for and click on the API you want, then click "Enable".
 
-.. code:: bash
+**Locally:**
 
-    export GOOGLE_CLOUD_PROJECT=$PROJECT_ID
-    export GOOGLE_APPLICATION_CREDENTIALS=$KEY_PATH
+    Install the desired :ref:`Python <python-installs>` and/or
+    :ref:`command-line <cli-installs>` tools.
+    Our tutorials demonstrate both methods;
+    you can choose the one you are most comfortable with.
 
-Enable the desired APIs. Here are some options:
+.. _python-installs:
 
-.. code:: bash
+Python API Installs
```

Comment:
Echoing zoom convo: Break section apart and move into introduction of the BigQuery, Pub SUb, and Cloud Storage pages.

Reply:
Leaving this for now since it all really is "initial setup" stuff and, if moved, "enable APIs" would be the only initial setup tasks not in this document. Can reconsider in the next round of edits.